.github/workflows/publish.yaml

```diff
@@ -41,3 +40,18 @@ jobs:
           tags: |
             ghcr.io/rode/ui:latest
             ghcr.io/rode/ui:${{ steps.gitextra.outputs.version }}
+
+      - name: Build Changelog
+        id: github_release
+        uses: mikepenz/release-changelog-builder-action@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Create Release
+        uses: actions/create-release@v1
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: ${{ github.ref }}
```

Comment:
Is there a way to get the tag name without the `refs/tags` prefix ending up in the release name?

Reply:
Good call. I removed those explicit references. We'll see what happens now.

Reply:
Done. It seems to work properly.